features/step_definitions/view_upcoming_events_steps.rb

```diff
@@ -21,3 +24,32 @@
 Then "I can see the event's name" do
   step "I can see \"#{@events[0].name}\""
 end
+
+When "I input a dummy keyword" do
+  fill_in "search", with: DUMMY_KEYWORD
+end
+
+Then "I press enter to search" do
+  find_field("search").native.send_key(:enter)
+end
+
+And "I can see that dummy keyword in the search box" do
```

Comment:
Consider use `"I can see that keyword in the search box"` the same for both cases no results and same results.